tests/accesscontrol/helper/helper.go

```diff
@@ -51,3 +53,35 @@ func DefineDeployment(replica int32, containers int, name string) (*v1.Deploymen
 
 	return deploymentStruct, nil
 }
+
+func SetServiceAccountAutomountServiceAccountToken(namespace, saname, value string) error {
```

Comment:
This function name might seem repetitive but we're setting the automountServiceAccountToken field on a ServiceAccount object, so it is accurate :joy: